privacy-deidentification/src/main/java/io/konig/privacy/deidentification/repo/DatasourceRepository.java

```diff
@@ -0,0 +1,88 @@
+package io.konig.privacy.deidentification.repo;
+
+import java.util.Random;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.util.StringUtils;
+
+import io.konig.privacy.deidentification.model.Datasource;
+import io.konig.privacy.deidentification.model.DatasourceResultSetExtractor;
+
+@Repository
+@Transactional
+public class DatasourceRepository {
+
+	@Autowired
+    JdbcTemplate template;
+	
+    public Datasource getByUuid(String uId){
+        String query1 = "select UID,ID,NAME,DE_IDENTIFICATION.DATASOURCE_NAME.LANGUAGE,DE_IDENTIFICATION.DATASOURCE_NAME.VALUE, DE_IDENTIFICATION.DATASOURCE_DESCRIPTION.Language AS DESCRIPTION_LANGUAGE ,DE_IDENTIFICATION.DATASOURCE_DESCRIPTION.VALUE as DESCRIPTION_VALUE, TRUST_LEVEL from DE_IDENTIFICATION.DATASOURCE INNER JOIN DE_IDENTIFICATION.DATASOURCE_NAME ON DE_IDENTIFICATION.DATASOURCE.UID=DE_IDENTIFICATION.DATASOURCE_NAME.DATASOURCE_UID  INNER JOIN DE_IDENTIFICATION.DATASOURCE_DESCRIPTION ON DE_IDENTIFICATION.DATASOURCE.UID=DE_IDENTIFICATION.DATASOURCE_DESCRIPTION.DATASOURCE_UID  and DE_IDENTIFICATION.DATASOURCE_DESCRIPTION.LANGUAGE=DE_IDENTIFICATION.DATASOURCE_NAME.LANGUAGE and UID=?";
```

Comment:
Move long query template to static fields, and format them so that they are more readable as shown below.

You don't need to worry about performance.  The string will be concatenated at compile-time (not at run-time).

```java
private static final String DATASOURCE_QUERY = 
   "select " +
      "UID," +
      "ID," +
      "NAME," +
      "DE_IDENTIFICATION.DATASOURCE_NAME.LANGUAGE," + 
      "DE_IDENTIFICATION.DATASOURCE_NAME.VALUE, " +
      "DE_IDENTIFICATION.DATASOURCE_DESCRIPTION.Language as DESCRIPTION_LANGUAGE , " +
      "DE_IDENTIFICATION.DATASOURCE_DESCRIPTION.VALUE as DESCRIPTION_VALUE," +
      "TRUST_LEVEL "
   "from " +
      "DE_IDENTIFICATION.DATASOURCE " +
      "inner join DE_IDENTIFICATION.DATASOURCE_NAME on" + 
         "DE_IDENTIFICATION.DATASOURCE.UID=DE_IDENTIFICATION.DATASOURCE_NAME.DATASOURCE_UID " +
      "inner join DE_IDENTIFICATION.DATASOURCE_DESCRIPTION on " +
         "DE_IDENTIFICATION.DATASOURCE.UID=DE_IDENTIFICATION.DATASOURCE_DESCRIPTION.DATASOURCE_UID" +  
       "and " +
          "DE_IDENTIFICATION.DATASOURCE_DESCRIPTION.LANGUAGE=DE_IDENTIFICATION.DATASOURCE_NAME.LANGUAGE " +
      "and " +
         "UID=?";
```